operator/apis/mlops/v1alpha1/pipeline_types.go

```diff
@@ -25,42 +25,77 @@ import (
 	"github.com/seldonio/seldon-core/apis/go/v2/mlops/scheduler"
 )
 
+//+kubebuilder:object:root=true
```

Comment:
**explain:** More conventional kubebuilder annotations to have this for both `struct`s.